### PR TITLE
loadout additions but better 

### PR DESCRIPTION
**Pull request by breadcrombs**
## About the Pull Request

Adds a few items into the loadout such as deodorant, mre, duct tape, etc

## Why It's Good For The Game

More items in the loadout for a better variety

## Changelog

:cl:
add: added mre's into general section of loadout
add: added point expensive duct tape into general section of loadout
add: added very point expensive combat knife into tactical section of loadout
code: changed the vegan mre description
/:cl:

### CODE REVIEW

**Comment by breadcrombs** [comment]
I'm open to point changes or removal of certain items

**Comment by User-U-U** [comment]
Idk for sure but I think the deodorant bottle can be emptied and used as an acid sprayer, would restrict from d class atleast

File: code/modules/client/preference_setup/loadout/lists/misc.dm [hunk 1]
```diff
@@ -58,11 +58,26 @@
...
+	cup["shot glass"] = /obj/item/reagent_containers/food/drinks/glass2/shot
+	cup["pint glass"] = /obj/item/reagent_containers/food/drinks/glass2/pint
+	gear_tweaks += new/datum/gear_tweak/path(cup)
+
 /datum/gear/coffeecup
 	display_name = "coffee cup"
 	path = /obj/item/reagent_containers/food/drinks/glass2/coffeecup
 	flags = GEAR_HAS_TYPE_SELECTION
-
+
```
Comment (Tsurupeta) [inline code/modules/client/preference_setup/loadout/lists/misc.dm]:
```suggestion

```

File: code/modules/client/preference_setup/loadout/lists/misc.dm [hunk 2]
```diff
@@ -293,3 +308,33 @@
...
+/datum/gear/tape
+	display_name = "duct tape"
+	description = "A roll of very strong tape."
+	path = /obj/item/tape_roll
+	cost = 6	
+	denied_roles = list(/datum/job/classd)
+
+/datum/gear/deodorant
+	display_name = "deodorant"
+	path = /obj/item/reagent_containers/spray/cleaner/deodorant
```
Comment (Tsurupeta) [inline code/modules/client/preference_setup/loadout/lists/misc.dm]: is there even a need for it?

Comment (breadcrombs) [reply code/modules/client/preference_setup/loadout/lists/misc.dm]: Deodorant is hard to get unless you are a class d from the mirror, it's a good rp item for foundation personnel. I could remove it if you want.

**Comment by breadcrombs** [comment]
deodorant has been nerfed and role restricted, do you guys want combat knife to take up all 10 points? I just want to have deodorant as a non class d

**Pull request merged**